docs/pages/reference/cli.mdx

```diff
@@ -1423,6 +1423,89 @@ DEBU [TSH]       Self re-exec command: tsh [status --format=json]. tsh/aliases.g
 ...
 ```
 
+#### Proxy Templates
```

Comment:
I agree with moving this out of OpenSSH guide but wouldn't this belong better under Server Access guides? E.g. Server Access / Guides / Proxy Templates?

Reply:
I feel like this reference will be more universally useful than a step by step guide, as it's more concise and universal. The exact way a customer uses it will be largely dependent on their trusted cluster configuration, e.g. how they name their nodes, proxy addresses, clusters, etc.

But we could also add a guide if we think that's useful. Should I just go through a specific trusted cluster scenario with a couple different leaf clusters with different naming conventions/needs, similarly to the [RFD](https://github.com/gravitational/teleport/blob/master/rfd/0062-tsh-proxy-template.md)?

@ptgott @alexfornuto Any thoughts on having a guide vs. cli reference vs. both? We can also copy the reference to the [using tsh guide](https://goteleport.com/docs/connect-your-client/tsh/), like how tsh aliases are in both locations.

Reply:
@Joerger I didn't mean to convert this to a step-by-step guide, I just meant to move under Server Access as-is.

Reply:
> @Joerger I didn't mean to convert this to a step-by-step guide, I just meant to move under Server Access as-is.

The guides in the docs now follow a more strict step by step format so I don't think that would be approved by the docs team.

Reply:
If `proxy_templates` is a section within the `tsh` configuration file, I think it makes sense to include this section there. The rest of the "tsh configuration files" H3 describes other sections in the file in a similar way, so this could be more discoverable than documenting all `tsh` config file sections here except for one.

On the other hand, I think the "tsh configuration files" section is pretty hard to find. Now that we have a "Connect your Client" section, I think we can rework the `/docs/connect-your-client/tsh` guide to include the whole "tsh configuration files" section.

I think we can split up this work so that:
- This PR adds a new section to "tsh configuration files" (without moving the section)
- A later PR, which either I or @alexfornuto can author, adds the "tsh configuration files" section to our tsh guide in the "Connect your Clients" docs section.

How does that sound?

Reply:
@r0mant Does this plan sound good to you?

Reply:
Sounds good.